addons/l10n_in/views/report_invoice.xml

```diff
@@ -2,9 +2,6 @@
 <odoo>
     <template id="l10n_in_report_invoice_document_inherit" inherit_id="account.report_invoice_document">
 
```

Comment:
can you remove this extra useless empty line as well plz?

Reply:
@smetl I have removed that empty line.